src/cmds/scripts/pbs_postinstall.in

```diff
@@ -806,7 +821,7 @@ create_home() {
 echo "*** PBS Installation Summary"
 echo "***"
 echo "*** Postinstall script called as follows:"
-echo "*** $0 $*"
+printf "*** $0 "; printf "%q " "$@"; printf "\n"
```

Comment:
Opened [PP-754](https://pbspro.atlassian.net/browse/PP-754) describing the issue.